checks.go

```diff
@@ -22,7 +23,6 @@ type Check struct {
 	Logo                 *string             `json:"logo"`
 	MailType             *string             `json:"mail_type"`
 	Memo                 string              `json:"memo"`
-	Message              *string             `json:"message"`
```

Comment:
Was this property deprecated from their API?

Reply:
ah good catch I meant to restore this